src/BizHawk.Client.Common/movie/bk2/Bk2Movie.HeaderApi.cs

```diff
@@ -52,7 +52,7 @@ public virtual bool StartsFromSavestate
 		public bool StartsFromSaveRam
 		{
 			// ReSharper disable SimplifyConditionalTernaryExpression
-			get => Header.TryGetValue(HeaderKeys.StartsFromSaveram, out var s) ? bool.Parse(s) : false;

```

Comment:
These should be reverted, as even though it's technically a "simplification" it makes the code less readable. The resharper disable is already evident of this.

Reply:
I disagree on this one, usually something is less readable/over complicated if you're returning true in the true condition of a ternary or likewise false in the false condition, but if someone is already intentionally ignoring the optimization then sure we can revert it